src/index.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
   typeMapper: require('./typeMapper'),
   attributeFields: require('./attributeFields'),
   simplifyAST: require('./simplifyAST'),
-  relay: require('./relay')
+  relay: require('./relay'),
+  jsonType: require('./types/jsonType')
```

Comment:
what do you mean by that?
Tested on my travis, and pass though...
[https://travis-ci.org/blueorbitz/graphql-sequelize/jobs/187616112](https://travis-ci.org/blueorbitz/graphql-sequelize/jobs/187616112)